lib/index.js

```diff
@@ -62,12 +74,11 @@ bedrock.events.on('bedrock-express.configure.routes', app => {
       store: ['validate', (results, callback) => {
         // generate new random ID for inbox
         const inbox = results.validate;
-        inbox.id = routes.inboxes + '/' + uuidV4();
-        const { owner: owner=req.user.identity.id } = inbox;
+        inbox.id = api.createInboxId();
+        const {owner: owner = req.user.identity.id} = inbox;
```

Comment:
Same thing here with no spaces around the equals (`=`). Again, open to discussion, but I was just following the style I had seen others writing ES6 in in various tutorials on the Web.